docs/rest-apis/algod/v2.md

```diff
@@ -133,15 +133,16 @@ Given a specific account public key, this call returns the accounts status, bala
 |Type|Name|Description|Schema|
 |---|---|---|---|
 |**Path**|**address**  <br>*required*|An account public key|string|
+|**Query**|**exclude**  <br>*optional*|When set to `all` will exclude asset holdings, application local state, created asset parameters, any created application parameters. Defaults to `none`.|enum (all, none)|
 |**Query**|**format**  <br>*optional*|Configures whether the response object is JSON or MessagePack encoded.|enum (json, msgpack)|
 
 
 **Responses**
 
 |HTTP Code|Description|Schema|
 |---|---|---|
-|**200**||[Account](#account)|
-|**400**|Malformed address|[ErrorResponse](#errorresponse)|
+|**200**|AccountResponse wraps the Account type in a response.|[Account](#account)|
+|**400**|Bad request|[AccountsErrorResponse](#accountserrorresponse)|
```

Comment:
I _think_ this is going to change with https://github.com/algorand/go-algorand/pull/3737